.travis.yml

```diff
@@ -119,6 +119,7 @@ script:
   - $make CFG_CRYPTO_{G,C}CM=n
   - $make CFG_CRYPTO_{MD5,SHA{1,224,256,384,512}}=n
   - $make CFG_WITH_PAGER=y
+  - $make CFG_WITH_PAGER=y CFG_TEE_CORE_DEBUG=y
```

Comment:
CORE_DEBUG should be 1 or 0. We may rewrite the following in source code

    #if defined(CFG_TEE_CORE_DEBUG) && CFG_TEE_CORE_DEBUG != 0